test/compatibility/test_fast_truncate_info.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import compatibility_test, os, shutil, sys
+
+class test_fast_truncate_info(compatibility_test.CompatibilityTestCase):
```

Comment:
I have a question, is this test runs under our regular compatibility tests as part of the evergreen?

Reply:
I'll double check and get back to you on this.

Reply:
@kommiharibabu No, this is currently not part of the regular compatibility tests on Evergreen. There is a [ticket](https://jira.mongodb.org/browse/WT-11101) that will involve explore and re-thinking our approach to compatibility testing and potentially trying to merge the existing compatibility_test_for_releases.sh tests with the test/compatibility tests. I'm wondering if that might be a better place to look at adding these tests to the regular compatibility testing.

Reply:
Understood. Thanks for the details.